tests/integration/targets/azure_rm_backupazurevm/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+- name: Fix resource prefix
+  set_fact:
+    resource_group: 'MyResourceGroup'
+    recovery_vault_name: 'MyRecoveryVault'
+    resource_id: "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachines/MyVM"
+    backup_policy_id: "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/microsoft.recoveryservices/vaults/testVault/backupPolicies/DefaultPolicy"
+
+- name: Enabling/Updating protection for the Azure VM
+  azure_rm_backupazurevm:
+    resource_group: "{{ resource_group }}"
+    recovery_vault_name: "{{ recovery_vault_name }}"
+    resource_id: "{{ resource_id }}"
+    backup_policy_id: "{{ backup_policy_id }}"
+    state: "create"
+  register: output
+
+- name: Assert that output has changed
+  assert:
+    that:
+      - output.changed
+
+- name: Trigger an on-demand backup for a protected Azure VM
+  azure_rm_backupazurevm:
+    resource_group: "{{ resource_group }}"
+    recovery_vault_name: "{{ recovery_vault_name }}"
+    resource_id: "{{ resource_id }}"
+    backup_policy_id: "{{ backup_policy_id }}"
+    state: "backup"
+  register: output
+
+- name: Assert that output has changed
+  assert:
+    that:
+      - output.changed
+      
+- name: Stop protection but retain existing data
+  azure_rm_backupazurevm:
+    resource_group: "{{ resource_group }}"
+    recovery_vault_name: "{{ recovery_vault_name }}"
+    resource_id: "{{ resource_id }}"
+    backup_policy_id: "{{ backup_policy_id }}"
+    state: "stop"
+  register: output
+
+- name: Assert that output has changed
+  assert:
+    that:
+      - output.changed
+
+- name: Get backup azure vm info
+  azure_rm_backupazurevm_info:
+    resource_group: "{{ resource_group }}"
+    recovery_vault_name: "{{ recovery_vault_name }}"
+    resource_id: "{{ resource_id }}"
+  register: output
+
+- name: Assert that output has changed
+  assert:
+    that:
+      - output.response.id != None
+      - output.response.name != None
+
+- name: Stop protection and delete data
+  azure_rm_backupazurevm:
+    resource_group: "{{ resource_group }}"
+    recovery_vault_name: "{{ recovery_vault_name }}"
+    resource_id: "{{ resource_id }}"
+    backup_policy_id: "{{ backup_policy_id }}"
+    state: "delete"
+  register: output
+  
+- name: Assert that output has changed
+  assert:
+    that:
+      - output.changed
+
```

Comment:
Please delete the last extra blank line！

Reply:
blank is removed from tasks/main/yml